effects/nixops/run-nixops.nix

```diff
@@ -110,6 +111,8 @@ mkEffect (
 
   getStateScript = ''
     stateFileName="$PWD/nixops-state.json"
+    '' +
+    lib.optionalString isStatefull ''
     getStateFile "$stateName" "$stateFileName"
     mkdir -p ~/.ssh
     getStateFile "$knownHostsName" ~/.ssh/known_hosts
```

Comment:
It's ok to download the state. That way if your network expression relies on, for example, valid ip addresses, it will still work.

Reply:
I removed this check. We indeed may need something like that in future as well :joy_cat: